### ISSUE TITLE
No internet connection dialog

### ISSUE THREAD

**Issue by Johann1982** [nicehash/NiceHashMiner-Archived#960]
Hi, I wrote about this previously but it is still an issue. Whenever there is a power failure or an internet related issue nicehash will just stop with the no internet connection dialog. Yes to continue without internet or no, exit. Is it not possible to change it so that it rechecks after 30seconds automatically and keep on retrying instead of waiting for user input? It is really frustrating as I have to log into all of my machines and manually close nicehash and restart. Only then does it continue mining. Please add this to the next build.

**Comment by imrecsoka** [comment]
Go to Local Area Connection Status and then click on Properties and then disable (uncheck) Internet Protocol Version 6.  Its a weird bug or as "funny"  , this works for me.
If you found this helpful, please say YAY or NAY

**Comment by Johann1982** [comment]
Nay.. It isn't a protocol issue. It is an issue when I have no internet connection and my machine reboots. Then it waits for user input. I would like it to automatically retry connecting after 30 seconds and keep retrying until internet is available.

**Comment by imrecsoka** [comment]
yeh sorry man i missed what u said above.
Maybe you can write a batch file which checks the process and if it is doing nothing kill it and restart.
but i mean u can surely start nicehashminer and then just the miners run because they automatically restart or resume when there is connection.
Powerfailure, maybe you have a bios which allows startup when there is one.
then in your start menu u add the nicehashminer to it but not sure how u click the start button
maybe just run the miners and forget nicehashminer

**Comment by fefsf** [comment]
Set up in task Scheduler to run nicehash at startup but delay the launch for a few minutes so that your internet can be established.

**Comment by imrecsoka** [comment]
yes but how does he click start automatically?

**Comment by fefsf** [comment]
He sets nhm to auto start mining

**Comment by Johann1982** [comment]
It does not matter if I schedule nicehash... The problem comes in when nicehash starts up without internet connection, which sometimes happens... Then the machine will just sit there and wait for me to click yes or no...  I would just like Nicehash to be able to re check for connection automatically and then start mining when it becomes available.

**Comment by unipsycho** [comment]
Yes I agree this is needed.  I have a rural internet and it drops at least once every couple days.  When this happens, nicehash pops up and waits for user to click continue, even though the internet has LONG recovered and could have been mining.  A way to suppress all popups would be best and keep nicehash retrying indefinitely for this exact reason.  Ridiculous to stall from a 1 minute internet outage, and very frustrating obviously.  I'll have to use a stand alone miner otherwise, as this is too costly.

**Comment by imrecsoka** [comment]
I just think the world should be a better place, for you and for me and the entire human race, there are people dying. If we care enough for the living , make it a better place for you and for me

**Comment by dacrypt** [comment]
I also request the dialog be removed and to put constant internet checks. The option to disable window dialog errors are indeed present but it's only for the miners. Just keep trying every 30s or so.

**Comment by sohachowdhury** [comment]
I disable the IPV6 but, It isn't working for me.
Please help.....

**Comment by imrecsoka** [comment]
Very strange man, mine was either because the nicehash website is busy...they are doing maintenance and it will show when u go to the web page.
The other was to enable IP4 and disable IP6.
Hmm, i hope u manage to get it to work.

**Comment by leomar-developer** [comment]
Is this really that difficult to solve? why in the world a mining software (which should be designed to work UNATTENDED BTW) requires the user to click on something when it loses internet connectivity? can't it just try again every few seconds until it can connect and resume mining? every decent mining software out there already does this. This issue should be a BLOCKER, you can easily lose an entire day of mining because of this critical bug. I don't understand why it has not been solved yet as I have seen people complaining about since 2016 (just take a look at #364), it could have been reported in other issues as well. This is one of the reasons I stopped using nicehash miner altogether and I bet a lot of other people did the same for this reason too!
